tests/languages/purescript/string_feature.test

```diff
@@ -0,0 +1,19 @@
+""
+"fo\"o"
+"foo \
+  \ bar"
+"foo -- comment \

```

Comment:
It's not really a comment, is it?

```suggestion
"foo -- comment lookalike \
```

Reply:
This one is a string, but results in an error because it is not closed.